instance/models/instance.py

```diff
@@ -139,17 +143,37 @@ def active_server_set(self):
         return self.server_set.exclude_terminated()
 
     @property
-    def status(self):
+    def current_server(self):
         """
-        Instance status
+        Current active server. Raises InconsistentInstanceState if more than
+        one exists.
         """
         active_server_set = self.active_server_set
         if not active_server_set:
-            return self.EMPTY
+            return
```

Comment:
@antoviaque  This property (current_server) is the result of the refactory of status and error properties and is meant to be used internally only by these two properties, i.e, it is not going to be returned to outside the models scope. I thought self.EMPTY is meant to be used by callsites outside models scope, but if that's not the case, I will revert this part.